packages/bento-design-system/stories/Components/TextField.stories.tsx

```diff
@@ -43,3 +44,15 @@ export const CustomAccessory = {
     rightAccessory: "👍",
   },
 } satisfies Story;
+
+export const NonDefaultType = {
+  args: {
+    value: "bento@example.com",
+    type: "email",
+  },
+  play: async ({ canvasElement }) => {
+    const canvas = within(canvasElement);
+    const textField = canvas.getByRole("textbox");
+    await expect(textField).toHaveAttribute("type", "email");
+  },
+} satisfies Story;
```

Comment:
what's this?

Reply:
I took it from your PR [here](https://github.com/buildo/bento-design-system/pull/816/files#diff-b6a90d9163fbce42877ebaeaa1166d69ae0d91c8fc69fc36a55f1c9518b104a9R48)